internal/webhookcert/ca_cert_provider.go

```diff
@@ -75,31 +77,36 @@ func (p *caCertProviderImpl) provideCert(ctx context.Context, caSecretName types
 }
 
 func (p *caCertProviderImpl) checkCASecret(ctx context.Context, caSecret *corev1.Secret) bool {
```

Comment:
Let's add more logs to this functions. We have to distinguish between 3 different situations:
1) No CA cert found - create new.
2) CA cert expired - create new.
3) Key size mismatch - create new.

It would be nice to have 3 distinct log messages.

Reply:
after discussion decided to skip this